exonum/src/blockchain/config.rs

```diff
@@ -359,7 +358,7 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(expected = "max_message_len (128) must be at least 330")]
+    #[should_panic(expected = "max_message_len (128) must be at least 354")]
```

Comment:
I'd leave it in the panic message because it's not user-friendly to tell that a configuration is not acceptable, but to not describe specific issues with it. However, I guess we may use the string `...must be at least"` in `should_panic` check which checks for a substring. In this way we won't 'leak' the implementation detail into the test.